distance_threshold: 0.01